pkg/cli/client.go

```diff
@@ -74,6 +77,37 @@ func doHTTPRequest(req *http.Request, verifyTLS bool, resultsPtr interface{}) (h
 		httpClient = createHTTPClient(verifyTLS)
 	}
 
+	// TODO check if /etc/con...etc exists
+
+	clientCert := "/etc/containers/certs.d/" + req.Host + "/client.cert"
```

Comment:
use fmt.Sprintf()

Reply:
I plan to use [path join](https://golang.org/pkg/path/#example_Join).
Is that ok?

Reply:
Yes